packages/circuit-ui/components/ImageInput/ImageInput.docs.mdx

```diff
@@ -4,17 +4,15 @@ import { Status, Props, Story } from '../../../../.storybook/components';
 
 <Status.Stable />
 
-An image input to handle image uploads in forms.
+The ImageInput component allows users to upload images, through drag and drop (desktop only), and local file browsing. It can be used on its own or as part of a form.
 
-<Story id="imageinput--base" />
+<Story id="components-imageinput--base" />
 
 <Props />
 
 ## Usage guidelines
 
-- **Do** …
+- **Do** use the Avatar component if the image input is read-only for the user.
 - **Do not** …
```

Comment:
A "Do" should be about the component itself, this sounds more like a "Don't" to me:

> 

```suggestion
- **Do** ...
- **Do not** use the ImageInput component if the image is read-only for the user.
```

Reply:
Done in 6817573